examples/asr/transcribe_speech.py

```diff
@@ -92,6 +93,9 @@ class TranscriptionConfig:
     # Recompute model transcription, even if the output folder exists with scores.
     overwrite_transcripts: bool = True
 
+    # Decoding strategy for CTC models
+    ctc_decoding: CTCCharDecodingConfig = CTCCharDecodingConfig()
```

Comment:
You may move creation of the ctc_decoding and rnnt_decoding inside the next if block and create them as needed.

Reply:
You mean the config? Both need to be present since one part of the config can be overridden by user (but it must already be there otherwise override will fail).

Reply:
I mean something like this?

`    if hasattr(asr_model, 'change_decoding_strategy'):
        # Check if ctc or rnnt model
        if hasattr(asr_model, 'joint'):  # RNNT model
		    rnnt_decoding: RNNTDecodingConfig = RNNTDecodingConfig(fused_batch_size=-1)
            asr_model.change_decoding_strategy(cfg.rnnt_decoding)
        else:
		    ctc_decoding: CTCDecodingConfig = CTCDecodingConfig()
            asr_model.change_decoding_strategy(cfg.ctc_decoding)`